landscaper/gardenlet/pkg/applier/gardenlet_chart.go

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) 2021 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package applier
```

Comment:
Hmm, why the pkg is called `applier`? I rather see inside a gardenlet landscaper component that does `Deploy` and `Destroy` (and uses a `kubernetes.ChartApplier` as many of our components do). I don't see a reason why applier is the name of this pkg.

Reply:
I renamed it to "chart"